### DISCUSSION TITLE
Upload to Edge Add-ons

### DISCUSSION THREAD

**Discussion by rustyx** [openstyles/stylus#1260] [Ideas]
Stylus is working perfectly fine in the new Chromium-based Edge, can even be installed directly from the Chrome Store.

So hereby an idea to upload it to the official [MS Edge Add-ons store](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home?hl=en-US).

**Comment by tophf** [comment]
It was suggested multiple times. Nothing's changed since #1198.
